playbooks/roles/jenkins/defaults/main.yml

```diff
@@ -29,20 +29,26 @@ dm_db_applications:
   - api
 
 jenkins_plugins:
+  - ansicolor
+  - blueocean
+  - build-monitor-plugin
+  - build-name-setter
+  - conditional-buildstep
+  - cvs
+  - extended-choice-parameter
   - git
   - github
   - github-oauth
-  - conditional-buildstep
-  - parameterized-trigger
-  - postbuildscript
-  - build-monitor-plugin
-  - nodejs
-  - ansicolor
-  - htmlpublisher
   - greenballs
-  - build-name-setter
-  - extended-choice-parameter
+  - htmlpublisher
   - lockable-resources
+  - nodejs
+  - parameterized-trigger
+  - postbuildscript
+  - rebuild
+  - translation
+  - workflow-aggregator  # aka 'Pipeline' suite
+  - ws-cleanup
```

Comment:
I'm 99% sure we don't need ws-cleanup - I think I accidentally installed that on our CI rather than my local instance when we were implementing VR tests. And then forgot to uninstall it. 🙈

Reply:
If you take it off the dashboard then I'll take it off this list 😸

Reply:
😆

Reply:
Just checked; it was installed November 28th, so it was definitely me. Thanks for removing. 👍